core-bundle/contao/templates/backend/be_ace.html5

```diff
@@ -61,26 +61,9 @@ window.ace && window.addEvent('domready', function() {
   // Disable command conflicts with AltGr (see #5792)
   editor.commands.bindKey('Ctrl-alt-a|Ctrl-alt-e|Ctrl-alt-h|Ctrl-alt-l|Ctrl-alt-s', null);
 
-  // Adjust the height of the editor
-  let updateHeight = function() {
-    let newHeight
-      = editor.getSession().getScreenLength()
-      * (editor.renderer.lineHeight || 16);
-    editor.container.style['height'] = Math.max(newHeight + 8, 30) + 'px';
-    editor.resize();
-  };
-
   editor.on('focus', function() {
     window.dispatchEvent(new Event('store-scroll-offset'));
-    updateHeight();
-  });
-
-  editor.getSession().on('change', function() {
-    ta.value = editor.getValue();
```

Comment:
Are you sure that we can remove this? It updates the textarea with the current editor value.

Reply:
You are right, I removed that by accident. :see_no_evil: Changed in fd4c21c again.